src/__test__/data.test.js

```diff
@@ -0,0 +1,12 @@
+import { countItem } from '../module/data.js';
+
+describe('ItemCounter', () => {
```

Comment:
[Optional] Consider adding a space between the test description to be "Meal Count"